formbuilder/utils.js

```diff
@@ -8,7 +8,7 @@ import config from "./config";
  * are stored. This is useful to always have one id to pass to the clients,
  * and they can figure out what the user token and collection name is.
  **/
-export function getUserToken(adminToken) {
+export function getFormID(adminToken) {
```

Comment:
If you change the name of the function, you need to update the description for it as well (returns the unique identifier of the form, from the administrative token)